.eslintrc.js

```diff
@@ -287,6 +291,12 @@ module.exports = {
         '@next/next/no-img-element': 'off',
       },
     },
+    {
+      files: ['docs/**/*'],
+      rules: {
+        '@stylistic/js/no-trailing-spaces': ['error'],
```

Comment:
Do we really need this when `prettier` is already removing trailing spaces?

Reply:
I had added this because it was turned on in `tslint` [here](https://github.com/mui/material-ui/pull/40020/files#diff-d3d90e3b411189c520aeca453b2aab5b62acf73ea98f22bef4908a69867370cbL6). But I think you are right, prettier will handle it.